### ISSUE TITLE
Aggregator Salle Odyssee event type mapping

### ISSUE THREAD

**Issue by troughc** [culturecreates/footlight-aggregator#51]
New information to add to the mapping file in Github for Tout Culture. 

New mapping is available for Salle Odysse event types here: https://docs.google.com/spreadsheets/d/15ri2MyKHGsbUcK8Kb83BijXMKkMZCfWjxcCK65Cmu_I/edit#gid=1544375604

Go to the Salle Odyssee tab. You will be mapping from keywords to Event types. cc @saumier @sahalali

**Comment by saumier** [comment]
@sahalali I tested with event "katherine levac" but it did not set the event type to humour. 

http://api.artsdata.ca/query?sparql=query_footlight_event_uri&uri=http://kg.footlight.io/resource/maisondelaculture-ca_humour-katherine-levac&frame=event_footlight&format=json&source=http://kg.artsdata.ca/culture-creates/footlight/maisondelaculture-ca

https://cms.footlight.io/dashboard/6308ef4a7f771f00431d939a/events/add-event/64af64493c19fa0064e76db6

**Comment by troughc** [comment]
@sahalali actually, I am not sure how to test this one. I looked at 'Cowboys Fringants' event and the Chanson event type wasn't listed. Is this because you added it after August 31? If so I can test again tomorrow after we import via the Aggregator.

**Comment by saumier** [comment]
Yes.

**Comment by troughc** [comment]
@sahalali I tested the event "Les soeurs boulay" id - 64af63a63c19fa0064e75341
The event type is Chanson (which should map from the keyword 'chanson'): PASSED
Several others also updated correctly: PASSED

Some issues as not all draft events are updating:

''Cowboys Fringants' event with the Chanson event type (id is 64af64643c19fa0064e7724a)
- The event type is Chanson (which should map from the keyword 'chanson'): this was not entered into event type (still indeterminé). This is because the event did not update yesterday (no change so the keywords didn't get mapped) 

 "Boucar Diouf" (64f02fe07d6aa8006434dfd0) did not get updated, 
- so the keyword 'humour" did not get added as event type Humour
- "Lise Dion": 64aff3993c19fa0064e7e60c  (humour)
"Christine Morency" (humour)


Can we force an update of all draft events so the keywords get mapped to event type? Or is that a bad idea? your opinion also welcome @saumier

**Comment by saumier** [comment]
@troughc If the event was updated and the event type mapped, then I would close this issue. 

In theory all upcoming events are updated even if there is no change in the data. So if the event's update date did not change then there is a glitch somewhere in the data pipeline.  

I looked in the logs and there are about 20 events from Salle Odyssée that couldn't de fetched from Artsdata because of a timeout in Artsdata. And about 10 events from Gatineau cloud. I will open another issue to look into these events to see why they are not getting through.



![Image](https://github.com/culturecreates/footlight-aggregator/assets/419491/9bab0388-2052-4589-8e1f-e6a3d3bebd77)
